### ISSUE TITLE
Feature Request: docker compose setup

### ISSUE THREAD

**Issue by magick93** [structureddynamics/Open-Semantic-Framework-Installer#14]
Hello

It would be great to have a docker compose setup that can run OSF.

**Comment by fgiasson** [comment]
@magick93 I agree, but at the moment Structured Dynamics does not have the resources to create that new feature. However if someone is willing to create it, we will support its development by testing it and integrating it into the OSF platform.

**Comment by magick93** [comment]
Does OSF require SELinux?

I have a docker compose project almost working, but it dies with:

> setenforce: SELinux is disabled

If SELinux is required then we cannot use docker, and that would be a great shame.

**Comment by fgiasson** [comment]
@magick93 no it is not, but in CentOS we had to set it to permissive. Probably that you are getting this error since `setenforce` tries to make it permissive. Probably that the installer should check if it is disabled before running that code. You can comment it here, and try again:

https://github.com/structureddynamics/Open-Semantic-Framework-Installer/blob/d13986bab0d7a82d06bdb0aea78b994800b22e66/inc/distros/OSFInstaller_CentOS_6.php#L32
